main_test.go

```diff
@@ -124,38 +132,50 @@ Terraform state outputs.
 | a_string | foo | string
 
 `,
-	}}
+	}, {
+		command: `cat testdata/show.json | ./terraputs`,
+		expectedOutput: `# Outputs
```

Comment:
certainly not the most idiomatic or elegant way to do this in Go. wiring up the cmd struct's `Stdin` to an `io.Reader` with an open handle on the statefile would be a more native method.  however, because you were already execing a compiled `terraputs` binary from within the tests, I thought this left us with nice descriptive examples of the various ways one might go about reading the statefile from the shell. it also happened to fit pretty well with the existing test table format, so i ran with it.

Reply:
Yessir - I think this is reasonable! And definitely feel free to PR any changes you think would make `terraputs` more Go-idiomatic, or better in general.

Reply:
@mdb thx for taking a look. 👍 in retrospect, i realize that my comment might not have come across as i intended. i'm not criticizing what you did as non-idiomatic, but sharing why i chose to stick with shell redirection when testing the stdin option rather than wiring up terraputs' stdin to the statefile natively in Go.